filebeat/input/kafka/config.go

```diff
@@ -106,14 +107,15 @@ var (
 // were chosen to match sarama's defaults.
 func defaultConfig() kafkaInputConfig {
 	return kafkaInputConfig{
-		Version:        kafka.Version("1.0.0"),
-		InitialOffset:  initialOffsetOldest,
-		ClientID:       "filebeat",
-		ConnectBackoff: 30 * time.Second,
-		ConsumeBackoff: 2 * time.Second,
-		WaitClose:      2 * time.Second,
-		MaxWaitTime:    250 * time.Millisecond,
-		IsolationLevel: isolationLevelReadUncommitted,
+		Version:           kafka.Version("1.0.0"),
+		InitialOffset:     initialOffsetOldest,
+		ClientID:          "filebeat",
+		ConnectBackoff:    30 * time.Second,
+		ConsumeBackoff:    2 * time.Second,
+		WaitClose:         2 * time.Second,
+		MaxWaitTime:       250 * time.Millisecond,
+		MaxProcessingTime: 20 * time.Microsecond,
```

Comment:
The docs say the default is 20s, but here we set Microsecond?

Reply:
🤦‍♂ yeap sorry for this. updated

Reply:
🤦‍♂ yeap sorry for this. updated